tests/testSed.py

```diff
@@ -139,12 +152,12 @@ def test_read_sed_flambda(self):
         Test how readSED_flambda handles the reading of SED filenames
         when we fail to correctly specify their gzipped state.
         """
-        scratch_dir = os.path.join(getPackageDir("sims_photUtils"),
-                                   "tests", "scratchSpace")
```

Comment:
if this is per test scratch space, why does it need to use a global setup_module scratch dir? We need the global if we are using scratch dir in class definitions. Maybe I'm missing some context on the PR. If only one test is using scratch space then this can be a mkdtemp call here.